deploy/src/main/sandbox-k8s/30-deploy-hono.sh

```diff
@@ -12,17 +12,18 @@
 # SPDX-License-Identifier: EPL-2.0
 #*******************************************************************************
 
-# This script deploys Hono to the Eclipse Foundation's sandbox at hono.eclipseprojects.io
+# This script deploys Hono with the configuration intended for the Eclipse Foundation's sandbox.
 # It requires that the namespace "hono" already exists and that the secret for the certificate is present.
 
-SCRIPTPATH="$(cd "$(dirname "$0")" && pwd -P)"
-source $SCRIPTPATH/common.sh
+set -ue
 
+SCRIPTPATH="$(cd "$(dirname "$0")" && pwd -P)"
 
-# add Helm repos
-helm repo add eclipse-iot https://eclipse.org/packages/charts $KUBECONFIG
-helm repo update $KUBECONFIG
+echo ""
+echo "Adding the Helm repository for Hono..."
+helm repo add eclipse-iot https://eclipse.org/packages/charts
+helm repo update
 
 echo ""
-echo "Deploying hono..."
-helm install eclipse-hono --dependency-update -f hono-values.yml -n hono eclipse-iot/hono $KUBECONFIG $HELM_WAIT
+echo "Deploying Hono..."
+helm install eclipse-hono --dependency-update -f $SCRIPTPATH/hono-values.yml -n hono eclipse-iot/hono --wait --timeout 5m0s
```

Comment:
why don't we use the `--create-namespace` and `--namespace` options here as well (as we do when installing cert-manager)? We do not seem to be deploying the certificates to namespace *hono*, though, are we?

Reply:
The certificates are indeed put into a secret in the namespace _hono_. You can find this configuration in the file `letsencrypt-certificate.yml`.

Reply:
I see